lib/network/modules/NodesHandler.js

```diff
@@ -144,7 +142,9 @@ class NodesHandler {
       throw 'Internal error: mass in defaultOptions of NodesHandler may not be zero or negative';
     }
 
-    util.extend(this.options, this.defaultOptions);
+    //this.options = {};
+    //util.extend(this.options, this.defaultOptions);  // Candidate for the Hall of Shame!
```

Comment:
Heh. For the epitaph, actually. Major stupidity was performed here. I do this often in my own code, so that I remember not to do that thing every time I encounter the comment.

But this is shared code, and keeping it in is just vanity. Hereby gone.